apps/web/cypress/fixtures/questionable-test-data.json

```diff
@@ -0,0 +1,171 @@
+{
+  "walkthroughs": [
+    {
+      "title": "doesn't successfully get to R11",
+      "steps": [
+        {
+          "question": "P01",
+          "type": "radio",
+          "answer": "yes"
+        },
+        {
+          "question": "P04",
+          "type": "radio",
+          "answer": "false"
+        },
+        {
+          "question": "P05",
+          "type": "radio",
+          "answer": "false"
+        },
+        {
+          "question": "P06",
+          "type": "checkbox",
+          "answer": "C"
+        },
+        {
+          "question": "P07",
+          "type": "radio",
+          "answer": "false"
+        },
+        {
+          "question": "P08",
+          "type": "radio",
+          "answer": "true"
+        },
+        {
+          "question": "P1",
+          "type": "radio",
+          "answer": "true"
+        },
+        {
+          "question": "P2",
+          "type": "radio",
+          "answer": "1B"
+        },
+        {
+          "question": "P3",
+          "type": "checkbox",
+          "answer": "1"
+        },
+        {
+          "question": "P6",
+          "type": "radio",
+          "answer": "true"
+        },
+        {
+          "question": "P9",
+          "type": "checkbox",
+          "answer": "1"
+        },
+        {
+          "question": "P11",
+          "type": "radio",
+          "answer": "3"
+        },
+        {
+          "question": "P12",
+          "type": "radio",
+          "answer": "false"
+        },
+        {
+          "question": "P13",
+          "type": "radio",
+          "answer": "false"
+        },
+        {
+          "question": "P17",
+          "type": "radio",
+          "answer": "true"
```

Comment:
After P17 `true` I'm getting R20 rather than going to P18. Which I believe is what we are confirming with this test that it "doesn't make it to R11"

Here are my answers after logging the current state after answering P17 and being on result R20 page.
![Screenshot 2024-07-05 at 10 07 16 AM](https://github.com/bcgov/House-Innovations-HOUSNAV/assets/7059124/4c9e495e-7e24-4dd9-809b-fbd3f50f0cbf)

Reply:
I added these flow examples to have a space to store the flows I think are giving an incorrect result. If we end up figuring out what the result really *should* be we can either update this data or run them as is to prove that our application has been fixed.